src/main/webapp/app/shared/layouts/profiles/page-ribbon.scss

```diff
@@ -36,8 +36,8 @@ Developement Ribbon
 
 /* top left */
 .ribbon-top-left {
-    top: -20px;
-    left: -20px;
+    top: 0px;
```

Comment:
Codacy complains about the units on the `0px` values. If that's best practice: consider editing our SCA config.

Reply:
Codacy is totally right. I missed to delete the `px`. 🙃